src/CTA.WebForms2Blazor/Extensions/CommentingExtensions.cs

```diff
@@ -0,0 +1,175 @@
+                    classDeclaration.OpenBraceToken.LeadingTrivia,
+                    SyntaxKind.OpenBraceToken,
+                    // For start, place after open brace
+                    SyntaxFactory.TriviaList(classDeclaration.OpenBraceToken.TrailingTrivia.Concat(triviaList))));
+            }
+            else
+            {
+                return classDeclaration.WithCloseBraceToken(SyntaxFactory.Token(
+                    // For end, place before close brace
+                    SyntaxFactory.TriviaList(classDeclaration.OpenBraceToken.LeadingTrivia.Concat(triviaList)),
+                    SyntaxKind.CloseBraceToken,
+                    classDeclaration.OpenBraceToken.TrailingTrivia));
+            }
+        }
+
+        public static ClassDeclarationSyntax AddClassBlockComment(
+            this ClassDeclarationSyntax classDeclaration,
+            string commentText,
+            int lineCharacterSoftLimit = Constants.DefaultCommentLineCharacterLimit,
+            bool atStart = true)
+        {
+            return classDeclaration.AddClassBlockComment(CommentWordWrap(commentText, lineCharacterSoftLimit), atStart);
+        }
+
+        private static IEnumerable<string> CommentWordWrap(string commentText, int lineCharacterSoftLimit)
+        {
+            var lines = new List<string>();
+            var words = commentText.Split(" ");
+            var currentLine = new StringBuilder();
+
+            foreach (var word in words)
+            {
+                if (string.IsNullOrEmpty(word))
+                {
+                    continue;
+                }
+
+                // Use greater than because right now there's an extra
+                // space at the end, though the exact line length isn't
+                // really a big deal anyway
+                if (currentLine.Length > lineCharacterSoftLimit)
+                {
+                    // Cut off extra space at the end
+                    lines.Add(currentLine.ToString().Trim());
+                    currentLine = new StringBuilder();
+                }
+
+                currentLine.Append($"{word} ");
+            }
+
+            if (currentLine.Length > 0)
+            {
+                lines.Add(currentLine.ToString().Trim());
+            }
+
+            return lines;
+        }
+
+        private static SyntaxTriviaList GetCommentLinesAsTriviaList(IEnumerable<string> commentText, int extraTabs = 0)
+        {
+            return SyntaxFactory.TriviaList(commentText.Select(commentLine =>
+                {
+                    var sb = new StringBuilder();
+
+                    for (int i = 0; i < extraTabs; i++)
+                    {
+                        // For some reason roslyn insists on
+                        // only using spaces so we use normal
+                        // spaces to keep form
+                        sb.Insert(sb.Length, " ", Constants.SpacesPerCommentTab);
+                    }
```

Comment:
This is fine as-is, but to clarify, my initial thought was that we could replace the loop with a single `sb.Insert()` call:
```csharp
sb.Insert(sb.Length, " ", Constants.SpacesPerCommentTab * extraTabs);
```

Reply:
Oh that makes a lot of sense, completely overlooked that. I'll make that change real quick